docs/reference/contracts/apis/managed.md

```diff
@@ -0,0 +1,294 @@
+                                  uint256 endTime,
+                                  uint256[] startWeights,
+                                  uint256[] endWeights)
+```
+
+Schedules a gradual weight change at the block timestamp provided in `startTime`, up until `endTime`. This is a permissioned function that can only be called by an authorized address set by the `owner`. If the pool is paused, the call will revert. Implemented in `ManagedPoolSettings`.
+
+## Joins and Exits
+
+### `getJoinExitEnabled`
+
+```solidity
+getJoinExitEnabled() 
+returns (bool)
+```
+
+Returns whether joins and exits are enabled. Implemented in `ManagedPoolSettings`.
+
+### `setJoinExitEnabled`
+
+```solidity
+setJoinExitEnabled(bool joinExitEnabled)
+
+emits JoinExitEnabledSet(bool joinExitEnabled)
+```
+
+Enables or disables joins and exits. This is a permissioned function that can only be called by an authorized address set by the `owner`. Implemented in `ManagedPoolSettings`.
+
+::: info
+**Note:** This does not affect Recovery Mode exits
+:::
+
+## Swaps
+
+### `getSwapEnabled`
+
+```solidity
+getSwapEnabled() 
+returns (bool)
+```
+
+Returns whether swaps are enabled. Implemented in `ManagedPoolSettings`.
+
+### `setSwapEnabled`
+
+```solidity
+setSwapEnabled(bool swapEnabled)
+
+emit SwapEnabledSet(bool swapEnabled)
+```
+
+Enables or disables trading. This is a permissioned function that can only be called by an authorized address set by the `owner`. Implemented in `ManagedPoolSettings`.
+
+## Allowlist
+
+### `getMustAllowlistLPs`
+
+```solidity
+getMustAllowlistLPs() 
+returns (bool)
+```
+
+Returns whether the allowlist for Liquidity Providers in enabled. Implemented in `ManagedPoolSettings`.
+
+### `isAddressOnAllowlist`
+
+```solidity
+isAddressOnAllowlist(address member) 
+returns (bool)
+```
+
+Returns whether a given address, `member`, is on the allowlist. Checks the allowlist regardless of whether the allowlist features is enabled. Implemented in `ManagedPoolSettings`.
+
+### `addAllowedAddress`
+
+```solidity
+addAllowedAddress(address member)
+
+emits AllowlistAddressAdded(address indexed member)
+```
+
+Adds an address, `member`, to the LP allowlist. This is a permissioned function that can only be called by an authorized address set by the `owner`. Implemented in `ManagedPoolSettings`.
+
+### `removeAllowedAddress`
+
+```solidity
+removeAllowedAddress(address member)
+
+emits AllowlistAddressRemoved(address indexed member)
+```
+
+Removes an address, `member`, from the LP allowlist. This is a permissioned function that can only be called by an authorized address set by the `owner`. Implemented in `ManagedPoolSettings`.
+
+### `setMustAllowlistLPs`
+
+```solidity
+setMustAllowlistLPs(bool mustAllowlistLPs)
+
+emit MustAllowlistLPsSet(bool mustAllowlistLPs)
+```
+
+Enables or disables the LP allowlist. This is a permissioned function that can only be called by an authorized address set by the `owner`. Implemented in `ManagedPoolSettings`.
```

Comment:
Add note that if someone has joined the pool and the LP allowlist is later turned on, they are still able to exit